dvc/command/repro.py

```diff
@@ -15,8 +15,7 @@
 class CmdRepro(CmdBase):
     def run(self):
         saved_dir = os.path.realpath(os.curdir)
-        if self.args.cwd:
```

Comment:
This is always initialized with os.curdir if not given a value. Alternative would be `if self.args.cwd != os.curdir`